data/json/items/gun/44paper.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "colt_army",
+    "type": "GUN",
```

Comment:
As I see. I suppose then the current reload and dispersion can be retained, especially as `proportional` does not get exact reload changes.